series-tmdb.en.yaml

```diff
@@ -2040,6 +2040,7 @@ entries:
         anilist-id: 150672
       - season: 2
```

Comment:
This should be season 1

Reply:
That's what I get for sending in a PR with 2 hours sleep, sorry about that